conftest.py

```diff
@@ -243,7 +297,16 @@ def _update_sensor_firmware():
 def reboot_sensor(make_valid_request):
     def _reboot_sensor():
         log.info("Send reboot request to sensor")
-        return make_valid_request(SensorMethod.REBOOT)
+        sensor_response = make_valid_request(SensorMethod.REBOOT)
+        result = "rebooting"
+
+        if "result" in sensor_response and isinstance(sensor_response["result"], dict):
```

Comment:
`reboot_sensor` doesn't return sensor info, so we shouldn't try to instantiate a `SensorInfo` object here. Since we don't have a class definition for what this sensor method returns, we just need to return `sensor_response["result"]`.

There's also no need to do an instance check on the value in `sensor_response`.